substrate/frame/support/src/traits/tokens/fungible/freeze.rs

```diff
@@ -121,3 +121,47 @@ pub trait Mutate<AccountId>: Inspect<AccountId> {
 		Self::set_frozen(id, who, a, Fortitude::Polite)
 	}
 }
+
+/// A vesting schedule over a fungible. This allows a particular fungible to have vesting limits
+/// applied to it.
+pub trait VestingSchedule<AccountId>: Sized {
```

Comment:
why is this moved?

Reply:
At a later date `Currency` is likely to be removed. At that point it's going to be all alone in the `traits::tokens::currency` namespace. Is there a better namespace for it to migrate to or should we leave it where it is for now even though it now takes a `Fungible`?